lib/node_modules/@stdlib/stats/base/dists/levy/quantile/manifest.json

```
@@ -0,0 +1,89 @@
+{
+    "options": {
+      "task": "build",
+      "wasm": false
+    },
+    "fields": [
+      {
+        "field": "src",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "include",
+        "resolve": true,
+        "relative": true
+      },
+      {
+        "field": "libraries",
+        "resolve": false,
+        "relative": false
+      },
+      {
+        "field": "libpath",
+        "resolve": true,
+        "relative": false
+      }
+    ],
+    "confs": [
+      {
+        "task": "build",
+        "wasm": false,
+        "src": [
+          "./src/main.c"
+        ],
+        "include": [
+          "./include"
+        ],
+        "libraries": [],
+        "libpath": [],
+        "dependencies": [
+          "@stdlib/math/base/napi/ternary",
+          "@stdlib/math/base/assert/is-nan",
+          "@stdlib/constants/float64/pinf",
+          "@stdlib/constants/float64/ninf",
+          "@stdlib/math/base/special/erfcinv",
+          "@stdlib/constants/float64/eps"
```

```suggestion
          "@stdlib/math/base/napi/ternary",
          "@stdlib/math/base/assert/is-nan",
          "@stdlib/math/base/special/erfcinv"
```
you need to remove the header files which are not in use.

---

do the same below as well.

---

builds failed there fore i added all this .